SofascoreAcademy/SofascoreAcademy/LeagueView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LeagueView.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 12.03.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+
+class LeagueView: BaseView {
+    
+    private let stackView = UIStackView()
+    private let leagueInfoView = LeagueInfoView()
+    private var matchViews: [MatchView] = []
+    
+    func update(data: [matchData]){
+        leagueInfoView.update(countryName: "Spain", leagueName: "LaLiga", leagueLogo: "leagueLogo")
```

Comment:
Na pocetku bi mozda trebalo isprazniti matchViews i stackView jer u slucaju da se poziva vise puta mislim da ce stari ostajati, a novi se extra dodavati, ali ne trebas to sad.